.github/workflows/stable-spec-tests.yml

```diff
@@ -8,7 +8,7 @@ on:
   workflow_dispatch:
 
 env:
-  FIXTURES_TAG: "verkle@v0.0.6"
+  FIXTURES_TAG: "verkle@v0.0.7-alpha-4"
```

Comment:
Whenever we merge this PR to `kaustinen-with-shapella`, I'd be able to create the final `v0.0.7`. That is because `execution-spec-tests` targets `kaustinen-with-shapella` by default, and I need this PR changes to be there first. 

Kind of the usual chicken and egg problems. I can do a later quick PR to change this to `v0.0.7` when that exists.